### PR TITLE
refactor olap property

### PR DESCRIPTION
**Pull request by javeme**
Change-Id: I137a42916eeee00775eafb55ee299a8a64e21671

### CODE REVIEW

**Review by zhoney — DISMISSED** [review]

File: hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java [hunk 1]
```diff
@@ -258,8 +258,13 @@ public void serverStarted(Id serverId, NodeRole serverRole) {
                  serverId, serverRole, this.name);
         this.serverInfoManager().initServerInfo(serverId, serverRole);
 
-        LOG.info("Search olap property key for graph '{}'", this.name);
-        this.schemaTransaction().initAndRegisterOlapTables();
+        // TODO: check necessary?
```
Comment (imbajin) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java:261]: we will rebuild missing olap table here (in another pr)

File: hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java [hunk 2]
```diff
@@ -258,8 +258,13 @@ public void serverStarted(Id serverId, NodeRole serverRole) {
...
+        LOG.info("Init olap property-keys for graph '{}'", this.name);
```
Comment (imbajin) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java:262]: seems the info is not accurate,  "Check olap property-keys and tables for xxx"  is better

File: hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java (hunk 1 above)
Comment (javeme) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java:261]: Ok, when will the table be missing?

Comment (imbajin) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java:261]: at least 2 case:
1. create property key success, create table failed (with any reason)
2. the olap table store file delete/broken by some operations

and once it happens, we can't delete or search or recovery it, this problem will fixed in that pr

**Review by imbajin — APPROVED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**